scripts/actorImporter.js

```diff
@@ -41,6 +41,10 @@ async function whatToDo(actorData, actorId) {
         label: game.i18n.localize('npcImporter.HTML.Override'),
         callback: async () => {
           log('Overriding existing Actor');
+          // Keep original images
+          if(actorOriginalData.img) actorData.img = actorOriginalData.img;
```

Comment:
`actorOriginalData`  does not exist in this context. We will need to pass it to the `whatToDo`... or do some refactoring around there... 🤔